spec/system/organization/user_views_an_organization_spec.rb

```diff
@@ -30,6 +30,16 @@
         end
       end
 
+      it "shows the right amount of articles in sidebar" do
```

Comment:
I'm not sure we need two tests here — the other is a super-set of what's being tested here.

Reply:
I think it's fine either way, but you could also condense them into a single example that tests different numbers of posts so it feels less arbitrary